added @mdsadique-inam/adonisjs-2fa package info

mdsadique-inam opened the pull request.

Julien-R44 commented:
Can you explain the differences with https://github.com/nulix-dev/adonis-2fa? Documentation looks exactly the same

I think adding this explanation of how your package differs from the one you forked would be interesting

mdsadique-inam commented:
Hi @Julien-R44 :wave: 
The package is similar because I forked it from there, but it is not the same, the difference is 
- The https://github.com/nulix-dev/adonis-2fa package has an incorrect implementation of config due to which whatever `issuer` is defined on `defineConfig` it always takes `App` as issuer, there is an open issue https://github.com/nulix-dev/adonis-2fa/issues/1 which is already a week old and also an MR resolving this issue but no response from package owner. The package itself is good but if its shows `App` on authenticator app instead of the real issuer then its a major issue for the client.
- Here I replaced `node-2fa` with `otplib` which gives the opportunity to customize the usage of the authenticator, like changing the digits of  top, or changing algorithm and others things also. I will add documentation on this later on.
- Instead of generating `qr` of chart.googleapis.com I create qr as an image blob because chart.googleapis.com is officially down. I did forgot to change the `qr` display on the documentation but I fixed that now.

Thank you

Julien-R44 commented:
Okay make sense. Thanks for taking the time to explain. I think it would be cool to include your explanation in your Readme, so that people can quickly see why they should use your package rather than the other one. It'll make things clearer on what is different
